core/error.go

```diff
@@ -83,9 +83,9 @@ func Errorf(f string, args ...interface{}) *Error {
 	return e
 }
 
-// ErrIfKeyExists checks if the error message contains "Key not found".
+// ErrIfKeyExists checks if the error message contains "key not found".
 func ErrIfKeyExists(err error) error {
-	if err == nil || strings.Contains(err.Error(), "Key not found") {
+	if err == nil || strings.Contains(err.Error(), "key not found") {
 		return nil
```

Comment:
@rchirakk: This was just a quick fix for us to be able to use `gofmt`, `go vet`, `golint` and the other tools on the entire code base. I intend to send some more PRs to refactor the error handling across the code base. I'd rather focus on starting the refactor work to avoid such issues in the future and make our code better.